src/temp/icons/LocationIcon.tsx

```diff
@@ -4,27 +4,25 @@ import { IconProps } from '../../features/common/types/common';
 const LocationIcon = ({ width, color }: IconProps) => {
   return (
```

Comment:
The LocationIcon was also used for `ContactDetails` - as a result of changing this icon, the UI has changed there as well. Be mindful of dependencies while updating existing components.